README.md

```diff
@@ -0,0 +1,70 @@
+active-directory-plugin
+================
+
+[![Build Status](https://jenkins.ci.cloudbees.com/job/plugins/job/active-directory-plugin/badge/icon)](https://jenkins.ci.cloudbees.com/job/plugins/job/active-directory-plugin/)
```

Comment:
Such embedded status is useless till the PR builder runs within the same job. Better to remove till we have multibranch